test/extensions/filters/http/jwt_authn/filter_config_test.cc

```diff
@@ -318,6 +320,43 @@ TEST(HttpJwtAuthnFilterConfigTest, RemoteJwksDurationVeryBig) {
                              HasSubstr("Duration out-of-range"));
 }
 
+TEST(HttpJwtAuthnFilterConfigTest, RemoteJwksInvalidUri) {
```

Comment:
Do we also want to test negative runtimefeature case (override it to false)?

Reply:
Sure, a test adding an override (using TestScopedRuntime).